Intersect.Server/Entities/Projectile.cs

```diff
@@ -42,6 +38,18 @@ public partial class Projectile : Entity
         public SpellBase Spell;
 
         public Entity Target;
+        
+        private static readonly int[,] ProjectileRotationDir =
+        {
+            { 0, 1, 2, 3, 4, 5, 6, 7 }, // Up
+            { 1, 0, 3, 2, 7, 6, 4, 5 }, // Down
+            { 2, 3, 1, 0, 7, 4, 5, 6 }, // Left
+            { 3, 2, 0, 1, 5, 6, 7, 4 }, // Right
+            { 4, 6, 7, 5, 2, 0, 3, 1 }, // UpLeft
+            { 5, 7, 4, 6, 0, 3, 1, 2 }, // UpRight
+            { 6, 4, 5, 7, 3, 1, 2, 0 }, // DownRight
+            { 7, 5, 6, 4, 1, 2, 0, 3 } // DownLeft
```

Comment:
https://github.com/AscensionGameDev/Intersect-Engine/pull/1708#discussion_r1126069711

also: de-duplicated the array and stored it over at ProjectileBase, where it can be used by both, the client and the server.